lib/modules/notifications/AndroidNotifications.js

```diff
@@ -91,4 +91,32 @@ export default class AndroidNotifications {
     }
     return Promise.resolve();
   }
+
+  deleteChannelGroup(groupId: string): Promise<void> {
+    if (Platform.OS === 'android') {
+      if (!(groupId instanceof string)) {
```

Comment:
Shouldn't this be `if (typeof groupId !== 'string') {`, instanceof string would throw `string undefined`?